packages/DropdownMenu/src/Confirmation/Confirmation.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Popover from "@paprika/popover";
+import ContentContainerStyled from "../ContentContainer.styles";
+import ConfirmationStyles from "./Confirmation.styles";
+
+const { func, string } = PropTypes;
+
+const propTypes = {
+  buttonSize: string,
+  confirmButtonType: string,
+  confirmLabel: string.isRequired,
+  description: string,
```

Comment:
Can we make this a `node`?  At some point someone will probably want to include a help link or something...